tensorrt_llm/executor/executor.py

```diff
@@ -204,6 +204,23 @@ def _get_next_client_id(self):
         self._last_client_id = (self._last_client_id + 1) & ((1 << 64) - 1)
         return self._last_client_id
 
+    def _get_logprob_params(self, request: GenerationRequest) -> Optional[dict]:
```

Comment:
A dedicated type for the logprob_params is preferred to help track this DS used across the codebase:

```python
class LogprobParams(Namedtuple):
    logprobs: Logprobs
    prompt_probs: Logprobs
    # Drop the logits once the logprobs are computed from the Worker
    drop_context_logits: bool = False
    # Drop the geneation_logits once the logprobs are computed from the Worker
    drop_generation_logits: bool = False
```

Reply:
updated